changefeedtrigger/blobEventsSample.py

```diff
@@ -0,0 +1,31 @@
+from azure.storage.blob import BlobServiceClient
+import azure.functions as func
+import random
+
+
+def create_event(blob_name, content, cont_client):
```

Comment:
I'd call it `simulate_account_activity` or so. It doesn't generate event per se. Events are side effect of this script.